src/Model/MediaInterface.php

```diff
@@ -28,7 +28,7 @@ interface MediaInterface
     public const MISSING_BINARY_REFERENCE = 'missing_binary_content';
 
     /**
-     * @return int|string|object|null
+     * @return int|string|null
```

Comment:
This is correct? What if I want to use a UUID?

Reply:
In admin-bundle they are not fully supported, and If you look closely at code here, you can't either, there are multiple places where $media->getId() is used directly as an array key.

Reply:
This is a discussion that needs to happen on whole sonata ecosystem, and I would love to support uuids everywhere (in fact I added this return previously. But the reality is that it does not support, and it is not only a matter of adding support here only. multiple changes are required on several bundles.

Reply:
Agree, if we want to support UUID an issue should first be opened on SonataAdmin because atm the phpdoc is not providing support and some code is using it as string|int.

Reply:
Issue opened: https://github.com/sonata-project/SonataAdminBundle/issues/7327